tests/test_barostat.py

```diff
@@ -80,10 +82,14 @@ def test_barostat_with_clashes():
     mol_a, _, _ = get_hif2a_ligand_pair_single_topology()
     ff = Forcefield.load_from_file("smirnoff_1_1_0_sc.py")
 
-    # Set the lambda to 0.0 and don't minimize, resulting in clashes in the system
-    unbound_potentials, sys_params, masses, coords, box = get_solvent_phase_system(
-        mol_a, ff, lamb=0.0, minimize_energy=False
-    )
+    # Construct water box without removing the waters around the ligand to ensure clashes
+    host_system, host_coords, box, host_top = build_water_system(3.0, ff.water_ff)
```

Comment:
nit: is it worth asserting that there actually are clashes here?

Reply:
Asserted in https://github.com/proteneer/timemachine/pull/1339/commits/2c68f3b702c6ef4618d26b1b04efac42cb6a5545